CRM/Lineitemedit/Form/Edit.php

```diff
@@ -197,12 +176,68 @@ public function submit($values) {
       $this->_id,
       $this->_lineitemInfo
     );
+
+    if (in_array($this->_lineitemInfo['entity_table'], ['civicrm_membership', 'civicrm_participant'])) {
+      $this->updateEntityRecord($this->_lineitemInfo);
+      $entity = strstr($this->_lineitemInfo['entity_table'], 'membership') ? 'membership' : 'event';
+      if (!$isTest) {
+        $this->ajaxResponse['updateTabs']['#tab_member'] = CRM_Contact_BAO_Contact::getCountComponent($entity, $contactId);
```

Comment:
I got a fatal error on this line. The values seem to have been passed to the function properly ('event', 12345) but it errored on: 
CRM/Contact/BAO/Contact.php(2726): CRM_Core_DAO::getFieldValue("CRM_Core_DAO_CustomGroup", NULL, "table_name")

One note -- the tab reference in this line needs to be conditioned on member or event (currently hardcoded to member)